.github/workflows/build-seas.yml

```diff
@@ -0,0 +1,130 @@
+name: Runtime Tests
+on:
+  workflow_call:
+env:
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
should we name this token for what it is so we can find it ?

Reply:
~It's an org token, for making requests to github. It's used in many places/repos~
This being a herodevs repo, I don't know anything about it.